docs/DeveloperGuide.md

```diff
@@ -196,80 +200,90 @@ Guarantees: Editing of internship is successful
   1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
   Use case resumes from step 1.
 
-### Use case: UC08 - Add an internship application
+### Use case: UC08 - Add an application
 
 #### MSS
 
-Guarantees: Adding of internship application is successful
+Precondition: User is on the Companies page and already has an existing list of internships of a company <br/>
```

Comment:
I noticed you changed of a company to in a company for others

Reply:
Okay, updated for consistency